### DISCUSSION TITLE
Upgrade my app to Next 13

### DISCUSSION THREAD

**Discussion by gunnerhowe** [vercel/next.js#49678] [Help]
### Summary

I have been trying to upgrade my NextJS app from version 12 to version 13 and I keep running into a lot of errors.

Would someone be willing to upgrade my app for me? I would love to assist, but I can't seem to get around all of the hydration errors I am experiencing.

**Comment by icyJoseph** [comment]
What kind of errors are you seeing?

You should only have to deal with `next/link` and its new behavior, and `next/image` new HTML structure.
